### ISSUE TITLE
Add you profile and help others

### ISSUE THREAD

**Issue by jsroyal** [jsroyal/JobProfile#1]
Just simple put details and PR.

**Comment by GeekyShiva** [comment]
hi, @jsroyal , Glad you came up with an awesome idea, I have made a PR, Please accept it and I have rehauled the overall look hope you will like it... I tried to make it more informative

**Comment by jsroyal** [comment]
Hi! @GeekyShiva Thanks for appreciating the idea.
Basically, the idea was originally something diff. and I expected PR from industry but people don't do that.
Ans I got all PR from students.
I had changed little bit idea which is you can see the in readme file

**Comment by GeekyShiva** [comment]
hey @jsroyal  I have made another PR with the changes that you made and if you need any changes and any other contributions to this please tell  me, I would be happy to make

**Comment by jsroyal** [comment]
@GeekyShiva Thanks! have you read RuleForRank?
Can you suggest something cooler! And have you got the understand the changed idea!

**Comment by GeekyShiva** [comment]
@jsroyal yes I read the RuleForRank and liked it as an idea but I suggest that weekly updating won't be possible, try to keep it fortnightly and you haven't mentioned anywhere your idea, please describe the same in readme so that I can even contribute for the same.

Also please review and accept the PR

**Comment by aman935** [comment]
@jsroyal  I have added the details to readme and made another pull request.

**Comment by jsroyal** [comment]
@GeekyShiva Cool!! but I think weekly will be good and only every Sunday before 11:59 pm people need to update after that I will merge into master.
Thanks, @aman935 will see you on the leaderboard.

I will do 
> Also please review and accept the PR

**Comment by himanshuk430** [comment]
Hello!
